driving_school/data/knowledge_article.xml

```diff
@@ -0,0 +1,287 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <template id="welcome_article_body">
+      <h1 class="oe-hint" data-oe-version="2.0">Driving School</h1>
+      <p>Hello there!</p>
+      <p>You just installed the Odoo for Driving School package. By doing so, we have installed a bunch of necessary apps to run your bar efficiently.</p>
```

Comment:
Is this line ok?
 we have installed a bunch of necessary apps to run your **bar** efficiently.